.github/workflows/python-bindings.yml

```diff
@@ -11,6 +11,11 @@ on:
       - '**/CMakeLists.txt'
       - 'libs/**'
       - '.github/workflows/python-bindings.yml'
+  workflow_run:
+    workflows: [ "pyfiction Docstring Generator" ]
+    branches: [ main ]
```

Comment:
Does this mean, this run only depends on `pyfiction Docstring Generator` when run on `main`? Is that desired?

Reply:
Yes correct, I first thought it will depend on PR's to `main`, but that does not make any sense.